todo-backend-springboot/pom.xml

```diff
@@ -62,6 +63,22 @@
             <artifactId>liquibase-core</artifactId>
             <version>${liquibase-core.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.atlassian.oai</groupId>
+            <artifactId>swagger-request-validator-core</artifactId>
```

Comment:
double check to make sure core is required.
if it is not required, double check the dependencyManagement section.